auth.py

```diff
@@ -16,12 +16,15 @@ def authenticate(url):
     s = requests.Session()
 
     # 1. Initiate sequence by querying the protected asset
-    s.get(url)
+    if("https://autolab.cs.cmu.edu/" in url):
+    	s.get(url, verify=False)
```

Comment:
Autolab seems to have their certificates misconfigured, if I leave out that argument I get an error: "SSL3_GET_SERVER_CERTIFICATE:certificate verify failed"